### DISCUSSION TITLE
How to add a Page Footer to a Word Document

### DISCUSSION THREAD

**Discussion by gjungb** [quarto-dev/quarto-cli#1946] [Q&A]
Hi,

I am trying to add a [Page Footer](https://quarto.org/docs/books/book-structure.html#page-footer) to a book which is rendered as `docx`, but the footer doesn't appear. Am I missing something?

Thx

**Comment by dragonstyle** [comment]
Right now, the page-footer is only supported for HTML output of books. It would be very nice to support in other formats as well - we will try to look at this for a future update.

In the meantime, if you were feeling ambitious you could write a filter that adds elements at the bottom of the document (or perhaps even make this a Quarto extension) to your liking...

**Comment by rcyost** [comment]
Any update on the header/footer? Thank you.

**Reply by mcanouil** [reply]
@rcyost you can see the issue opened to track this on the right (in desktop view).

- #1948
